src/modules/shared/components/CardList/CardList.module.scss

```diff
@@ -0,0 +1,11 @@
+@use "styles/abstract" as *;
+
+.c-card-list {
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(230px, 1fr));
+
+    .c-card-list_card {
```

Comment:
Volgens de bem-notatie moet dit `.c-card-list__card` zijn. Kan je ook als `&__card` schrijven in dit geval.